src/plugin/Plugin.cpp

```diff
@@ -317,17 +238,14 @@ void Plugin::loadScript() {
     }
 }
 
-/**
- * Execute lua function
- */
-bool Plugin::callFunction(string fnc) {
-    lua_getglobal(lua, fnc.c_str());
+auto Plugin::callFunction(const std::string& fnc) -> bool {
+    lua_getglobal(lua.get(), fnc.c_str());
 
     // Run the function
-    if (lua_pcall(lua, 0, 0, 0)) {
-        const char* errMsg = lua_tostring(lua, -1);
-        map<int, string> button;
-        button.insert(std::pair<int, string>(0, _("OK")));
+    if (lua_pcall(lua.get(), 0, 0, 0)) {
+        const char* errMsg = lua_tostring(lua.get(), -1);
+        map<int, std::string> button;
```

Comment:
`std::map` here too.

Reply:
done, also added a todo in our metaissue